utils.sh

```diff
@@ -33,6 +34,11 @@ function custom_ntp(){
     done
     NTPFILECONTENT=$(echo "${NTPFILECONTENT}" | base64 -w0)
     sed -i -e "s/NTPFILECONTENT/${NTPFILECONTENT}/g" assets/generated/*-chronyd-custom.yaml
+    IGNITION_VERSION=$(yq -r .spec.config.ignition.version ${ASSESTS_DIR}/99_openshift-machineconfig_99-master-ssh.yaml)
```

Comment:
CI can't seem to find yq, although we do install it: https://github.com/openshift-metal3/dev-scripts/blob/2dc2c365c3fe460a3fbd64e6413641e5168d055d/01_install_requirements.sh#L36

Is it not in $PATH?

Reply:
I guess not, it seems we're running dev-scripts as root:

```
+(./01_install_requirements.sh:36): main(): pip3 install --user 'yq>=2.10.0'
WARNING: Running pip install with root privileges is generally not a good idea. Try `pip3 install --user` instead.
Collecting yq>=2.10.0
```

And AFAICS that doesn't include $HOME/.local/bin in the default path.

It would be preferable to run the scripts as a regular user with sudo NOPASSWD - @andfasano do you recall if there's a reason we can't do that?

As an alternative I guess we can either append to the PATH or explicitly specify it for yq?

Reply:
Packet defaults to giving you the root user. We'd probably have to pass cloudinit user data to configure a separate user.

Reply:
> As an alternative I guess we can either append to the PATH or explicitly specify it for yq?

Adding it to $PATH sounds fine for now.

Reply:
We've always used for simplicity root on Packet, also for the Ansible playbook. Adding to $PATH seems fine also for me.

Reply:
Ok I'll add it to the PATH for now - IMO it would be nice if we could move CI to use a non-root user, since we expect most users to deploy like that (and we could miss stuff like a missing sudo in CI when it all runs as root).

I guess we could just add a user via the ansible setup tasks before running dev-scripts, would that be simpler than the cloud-init approach?

Reply:
That would be the ideal solution, since it's better to distinguish between the user that configure the Packet instance (root via Ansible) and the user that consumes the dev-script installation (ie du-user).
Other administrative actions could remain in the scope of root user (ie copying artifacts back for the CI)

Reply:
Ok for now I added `$HOME/.local/bin` to the path